### ISSUE TITLE
[amplify-appsync-simulator] "Cannot use GraphQLSchema from another module or realm." Error when trying to use in project

### ISSUE THREAD

**Issue by tqhoughton** [aws-amplify/amplify-category-api#692]

**Comment by ykethan** [comment]
Hello @tqhoughton, Thank you for reaching out. As you have pointed out this is not intended to consume as a package outside of Amplify. Marking this a feature request.

**Comment by mkossoris** [comment]
Interested in this feature as well. There's no other good alternative for running vanilla non-framework (Serverless Framework, Amplify, etc) AppSync API projects locally. That said, I believe this issue should probably be located in the amplify-cli project for proper visibility and support as [that is where the simulator package is located](https://github.com/aws-amplify/amplify-cli/tree/dev/packages/amplify-appsync-simulator) right?

**Comment by dylanfprice** [comment]
I am interested in this feature as well.

**Comment by tqhoughton** [comment]
Yeah I'm not sure why this was moved out of the amplify-cli project
